mps_youtube/commands/play.py

```diff
@@ -7,8 +7,8 @@
 from . import command, WORD, RS
 from .songlist import plist
 from .search import yt_url, related
-from ..player import play_range
-
+from ..players.mpv import Mpv
+from ..players.mplayer import Mplayer
```

Comment:
We will apply a fix for that as well and once it is ready we can discuss it further.